.eslintrc

```diff
@@ -106,7 +106,8 @@
         "excludedForTesting.js"
       ],
       "rules": {
-        "oppia/e2e-action": "error"
+        "oppia/e2e-action": "error",
+        "oppia/nested-awaits": "error"
```

Comment:
Any specific reason for adding this check here? (Note: This will exclude running the checks in the files mentioned above!)

Reply:
It is mentioned on the issue page to put it here.
https://github.com/oppia/oppia/issues/11496

On the 3rd part of the 2nd point.
Add your rule to to the object with `oppia/e2e-action` in `overrides `of `.eslintrc`

Reply:
Yeah, it's fine to put this check here. We plan to first get these checks written and working on a few files before expanding to the rest of the code base. That way new contributors don't have to fix everything in their first PR